workshops/discord_message_bot/README.md

```diff
@@ -0,0 +1,395 @@
+---
+name: 'Discord Custom Message Bot'
+description: 'Save your own own custom messages with a Discord Bot!'
+author: '@JakeGerber'
+---
+
+# Create a Discord Bot that allows you to save custom messages!
+
+<img src="https://cloud-bj4vorj8t.vercel.app/examplebot.png" width="380" alt="Message Example">
+
+## Bot Setup
+
+Let's set up a bot through Discord before creating its features! Click [here](https://discord.com/developers/applications).
+
+Click the "New Application" button at the top right.
+<img src="https://cloud-noinx6mnk.vercel.app/0screenshot__1379_.png" width="900" alt="New Application Button">
+
+This creates a new bot where you can customize the name, description, and profile picture!
+<img src="https://cloud-jlzsnh85x.vercel.app/0screenshot__1380_.png" width="900" alt="Bot Profile">
+
+Now click the "Bot" tab on the right side of your screen.
+<img src="https://cloud-e5r0obhgo.vercel.app/0screenshot__1381_.png" width="900" alt="Bot Side Bar">
+
+Click "Add Bot" to generate a bot token! This identifies the bot! Give it to nobody!
+<img src="https://cloud-lix7k1shj.vercel.app/0screenshot__1382_.png" width="900" alt="Bot Token">
+
+<img src="https://media4.giphy.com/media/pvl3qUsgblNOo/200.gif" width="380" alt="Zelda Gif">
+
+## Repl.it Setup
+
+We're going to use [Repl.it](https://repl.it/~) to host the bot. It is an online IDE that makes it easy to setup and run the bot!
+
+Create a new repl and use Node.js as the language.
+
+<img src="https://cloud-otu0relhe.vercel.app/0screenshot__1383_.png" width="600" alt="Node.js Repl">
+
+
+Let's start creating it's functions!
+
+<img src="https://media1.tenor.com/images/d17514e2c03ec6b0e67ba7f18439a011/tenor.gif" width="380" alt="Shawn Sheep Gif">
+
+First, create two files called index.js and msgs.json. 
+
+  - Your messages will be stored in msgs.json. 
+  
+  - JSON is a file format that allows you to store data as a JavaScript object (key/value pair). If you want to learn more, [check here](https://www.json.org/json-en.html). 
+  
+  - As with defining a normal JavaScript object, your .json file should contain brackets as shown below.
+  
+
+<img src="https://cloud-gh7l7h2q1.vercel.app/json_example.png" width="380" alt="Write Command Example">
+
+In your index.js file, you will start off with code that looks like this. It will be explained below.
+
+```js
+const Discord = require('discord.js')
+const client = new Discord.Client()
+const fs = require('fs')
+client.msgs = require('./msgs.json')
+prefix = '!'
+
+client.once('ready', () => {
+  console.log('Ready!')
+})
+
+client.on('message', (message) => {})
+
+client.login('token')
+```
+
+_Make sure you insert your bot token where 'token' is!_
```

Comment:
Wait but repl.it code is public, so if someone puts their token in the code, they're sharing it with everyone who looks at the code. May be worth using and explaining .env